src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -945,6 +948,21 @@ private ConfigComparisonResult compareConfigs() {
     return comparisonResult;
   }
 
+  private void askUserForName() {
+    TextInputDialog dialog = new TextInputDialog();
+    dialog.setTitle("Enter your name");
+    dialog.setHeaderText("For auditing purposes, enter the name(s) of everyone currently "
+        + "operating this machine.");
+    dialog.setContentText("Name:");
+    Optional<String> result = dialog.showAndWait();
+
+    if (result.isPresent()) {
+      Logger.info("Name(s) of operators, as entered interactively: " + result.get());
```

Comment:
Note that, a few lines below this in the logs, it also gives:

```
2023-06-26 14:49:56 PDT INFO: Computer name: abc
2023-06-26 14:49:56 PDT INFO: User name: xyz
```

This could be confusing to people. Maybe at the very least we should change "user name" to "Computer user name" or something?